wiki/Modding/Star_priority/fr.md

```diff
@@ -1,21 +1,18 @@
 ---
 stub: true
-outdated: true
-outdated_since: 78d85860fc09d0ab7b6210300259c8e488b540ea
 ---
 
-<!-- term definition is unused now that forum-based modding is gone -->
-
 ### Star priority
 
-Le terme **star priority** a deux sens:
+Le terme **star priority** a deux sens :
 
 1. Une méthode pour aider les Beatmap Nominators à trouver des beatmaps dignes de recevoir une bubble et d'être classées ultérieurement.
    - La star priority d'une beatmap est déterminée par la somme du nombre de messages de mods qui ont obtenu des kudosu dans le fil de discussion de la beatmap et du total étoiles de kudosu que la beatmap a reçues.
    - Une étoile de kudosu peut être donnée par n'importe quel utilisateur à n'importe quel beatmap, chaque étoile coûtant un kudosu.
    - Le taux de conversion était auparavant de trois kudosu pour une étoile kudosu qui augmentait la star priority de deux.
-   - Plus la star priority était élevée, plus le fil de discussion de la beatmap apparaissait haut dans le sous-forum Work in Progress ou Pending.
+   - Plus la star priority était élevée, plus le fil de discussion de la beatmap apparaissait haut dans le sous-forum Work in Progress ou En attente.
```

Comment:
Plus la star priority est élevée, plus le fil de discussion de la beatmap apparaît haut dans le sous-forum Work in Progress ou En attente.